templates/satellite.yaml

```diff
@@ -0,0 +1,36 @@
+---
+:name: Satellite
+:description: Install Satellite server
+:enabled: true
+:log_dir: /var/log/foreman-installer
+:store_dir: ''
+:log_name: satellite.log
+:log_level: DEBUG
+:no_prefix: false
+:mapping: {}
+:answer_file: {{ satellite_answers_file_destination }}
+:installer_dir: /usr/share/katello-installer-base
+:module_dirs:
+- /usr/share/foreman-installer/modules
+- /usr/share/katello-installer-base/modules
+:colors: true
+:color_of_background: :dark
+:hook_dirs:
+- /usr/share/katello-installer-base/hooks
+:custom: {}
+:low_priority_modules: []
+:verbose_log_level: info
+:default_values_dir: /etc/foreman-installer/scenarios.d
+:hiera_config: /usr/share/foreman-installer/config/foreman-hiera.conf
+:order:
+- certs
+- foreman
+- katello
+- foreman_proxy
+- foreman_proxy::plugin::pulp
+- foreman_proxy_content
+- puppet
+:parser_cache_path:
+- /usr/share/foreman-installer/parser_cache/foreman.yaml
+- /usr/share/katello-installer-base/parser_cache/katello.yaml
+:password: X_ZAhOZSLf_-ZLoGWBOydqFKH52SC76rNLo3od0zp_g
```

Comment:
I have absolutely no idea. It was auto-populated by the yum installer, so I would have to look at the RPM build spec to know if it was generated at RPM install time or is just a hard-coded default.

Reply:
Similar to the password comment above, I'd just like to make sure that it's not easy for this role to be used and subsequently a default/non-secure password created for deployments.

Reply:
So this seems to be auto-generated by the RPM at install time. I'm going to switch away from templating and use the dreaded `lineinfile` mechanism to edit the file that's been created instead of having this coded into the role somewhere.